### DISCUSSION TITLE
img metadate

### DISCUSSION THREAD

**Discussion by HonZuna** [lllyasviel/Fooocus#1454] [Q&A]
Hello, I recently switched from Auto1111 to Fooocus for most of my work, and it's an amazing tool. I practically have only one question: is there a way to enable the saving of prompts and settings into the metadata of images so that it's possible to find out what prompt was used later? I often return to my older creations and this way I lose the ability to continue or remind myself how I achieved that particular result. I can imagine that some people might not want this, but why shouldn't there at least be an option to enable it? As far as I know, all other tools have this enabled by default.

PS: I heard about private log .txt files but i have no idea how to enable it.

**Comment by usigg3** [comment]
The log files in the output directory contains some data like the prompt and the seed but nothing about the image prompts or similar... it's really frustrating when my computer has to restart or the browser crashes....

**Comment by xhoxye** [comment]
Fooocus Private Log https://github.com/lllyasviel/Fooocus/discussions/160

Fooocus is committed to protect user privacy and will never write any hidden data in your generated images。

The option “write hidden meta data” does not even exist in Fooocus, so that you do not need to constantly check if you have forget to turn it on/off, or constantly using some tools to check it.

Once you are in Fooocus, you can just trust that everything is absolutely private and clean without any hidden things.

Advanced, Setting, History Log
![QQ截图20231217093537](https://github.com/lllyasviel/Fooocus/assets/129571231/334b17c4-b8df-4d12-833f-46ad80b8bed7)

![QQ截图20231217093717](https://github.com/lllyasviel/Fooocus/assets/129571231/2b91f43a-23df-4bc9-969a-62353b4d3fe3)
